index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Handcrafted, home-made masterpieces</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
```

Comment:
It is not recommended to add blank lines without a reason. Adding blank lines without a reason violates HTML best practices.